test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/ReadWrite/Updating/Resources/UpdateToOneRelationshipTests.cs

```diff
@@ -0,0 +1,109 @@
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using JsonApiDotNetCore.Serialization.Objects;
+using Xunit;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.ReadWrite.Updating.Resources
+{
+    public sealed class UpdateToOneRelationshipTests
+        : IClassFixture<IntegrationTestContext<TestableStartup>>
+    {
+        private readonly IntegrationTestContext<TestableStartup> _testContext;
+
+        public UpdateToOneRelationshipTests(IntegrationTestContext<TestableStartup> testContext)
+        {
+            _testContext = testContext;
+        }
+        
+        [Fact]
+        public async Task Cannot_create_OneToOne_relationship_from_principal_side()
```

Comment:
Earlier remarks apply here too: test name, fakers, related resource creation. A single test is sufficient in this file.